common/check_page_status/check_page_status.py

```diff
@@ -0,0 +1,44 @@
+from urllib.request import Request, urlopen
+import json
+import requests
+import argparse
+import logging
+
+
+def get_sitemap(url):
+    req = Request(url + 'sitemap/?format=json')
```

Comment:
is there any specific reason you aren't using `requests.get()`

Reply:
Will do.